Core/src/org/sleuthkit/autopsy/datamodel/BlackboardArtifactNode.java

```diff
@@ -476,10 +493,17 @@ public BlackboardArtifact getArtifact() {
          * If the source content of the artifact represented by this node is a
          * file, add an action to view the file in the data source tree.
          */
-        AbstractFile file = getLookup().lookup(AbstractFile.class
-        );
-        if (null != file) {
-            actionsList.add(ViewFileInTimelineAction.createViewSourceFileAction(file));
+        if (this.srcContent instanceof AbstractFile) {
+            actionsList.add(ViewFileInTimelineAction.createViewSourceFileAction((AbstractFile) this.srcContent));
+        } else if (this.srcContent instanceof DataArtifact) {
+            try {
+                if (ViewArtifactInTimelineAction.hasSupportedTimeStamp((DataArtifact) this.srcContent)) {
```

Comment:
The cast to a is too strong, change to BlackboardArtifact if needed.  Both hasSupportedTimeStamp and ViewArtifactInTimelineAction take BlackboardArtifact